Core/DISET/private/Transports/BaseTransport.py

```diff
@@ -25,7 +25,7 @@
 
 import time
 import select
-import cStringIO
+from six import BytesIO
```

Comment:
I didn't check, but when `StringIO` or BytesIO`?

Reply:
It's analogous to opening a file with `open(filename, mode="wt")` and  `open(filename, mode="wb")` though Python 2 doesn't do a very good job teaching you to think about the difference.

* `StringIO` should be used when you want something you would open in a text editor: XML files, JDL, HTML.
* `BytesIO` should be used when you just want a bag of bytes such as a compressed tarfile or an image.

I think I've got these correct but we'll only know for sure when we start executing tests. I have a draft branch locally which gets 80% of them passing so I'll make a PR to start executing tests as soon as this PR is merged.